### PR TITLE
Updated home page march 8th 

### PR DESCRIPTION
**Pull request by anikapatychuk**
Screenshot 2024-03-08 090854

### CODE REVIEW

File: mobile/voice_journal_app/lib/home.dart [hunk 1]
```diff
@@ -1,43 +1,71 @@
 import 'package:flutter/material.dart';
+import 'theme.dart';
 
 void main() {
   runApp(
     MaterialApp(
       home: Scaffold(
```
Comment (pabloj2001) [inline mobile/voice_journal_app/lib/home.dart]: To turn this into a class, we'll want to place everything in the `Scaffold` in the `build` function of the class, just like it is right now in the `MyHomePage` class in the `main.dart` file

**Review by pabloj2001 — CHANGES_REQUESTED** [review]
Love how it's looking so far! The only thing I want to suggest is turning this home page into a class (StatelessWidget) and using it from `main.dart` so we only have one entry point to the app. To do this we'd just have to turn the `main` function in this file into a new class, then delete the `MyHomePage` class in `main.dart` and replace
```
home: const MyHomePage(title: 'Flutter Demo Home Page'),
```
in the `MyApp` class with our new Home class. Let me know if you have any questions about this!

File: mobile/voice_journal_app/lib/main.dart [hunk 2]
```diff
@@ -8,34 +9,29 @@ class MyApp extends StatelessWidget {
...
+        // Add more theme customization as needed
+        useMaterial3: true, // Opt-in to use Material 3 features
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: HomePage(),
     );
   }
 }
-
 class MyHomePage extends StatefulWidget {
```
Comment (pabloj2001) [inline mobile/voice_journal_app/lib/main.dart]: We can remove this class now since it's no longer used.

**Review by pabloj2001 — CHANGES_REQUESTED** [review]
Great job! Just one last suggestion and then I think we can merge 👍

File: mobile/voice_journal_app/lib/main.dart [hunk 3]
```diff
@@ -8,52 +9,29 @@ class MyApp extends StatelessWidget {
...
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
-
-  final String title;
-
-  @override
-  State<MyHomePage> createState() => _MyHomePageState();
-}
-
 class _MyHomePageState extends State<MyHomePage> {
```
Comment (pabloj2001) [inline mobile/voice_journal_app/lib/main.dart]: We should remove this class as well since it was only used in `MyHomePage` which we just deleted.

**Review by pabloj2001 — CHANGES_REQUESTED** [review]
Very last thing, we should have deleted `_MyHomePageState` along with deleting `MyHomePage` since it isn't used anywhere else

**Review by Alec-Hallman — APPROVED** [review]
Looks great!

**Review by pabloj2001 — APPROVED** [review]
Looks Good to Me! 👍

**Pull request merged**